From b5f7cd98fbfb8cee5cbc38d57b65c6a0a7e9e80c Mon Sep 17 00:00:00 2001
From: "jie.he" <jie1.he@intel.com>
Date: Thu, 18 Jul 2019 18:34:06 +0800
Subject: [PATCH] update configure.ac to recognize more linux OS alias

---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 2c4d2888d2..f29cb7f2d9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2319,14 +2319,14 @@ AS_CASE(["$rb_cv_fiber_coroutine"], [yes|''], [
 	[x*64-darwin*], [
 	    rb_cv_fiber_coroutine=amd64
 	],
-	[x*64-linux], [
+	[x*64-linux*], [
 	    AS_CASE(["$ac_cv_sizeof_voidp"],
 	        [8], [ rb_cv_fiber_coroutine=amd64 ],
 	        [4], [ rb_cv_fiber_coroutine=x86 ],
 		[*], [ rb_cv_fiber_coroutine= ]
 	    )
 	],
-	[*86-linux], [
+	[*86-linux*], [
 	    rb_cv_fiber_coroutine=x86
 	],
 	[x64-mingw32], [
-- 
2.17.1

